fbw-a32nx/src/wasm/systems/a320_systems/src/air_conditioning.rs

```diff
@@ -272,18 +272,30 @@ impl A320AirConditioningSystem {
                     context,
                     AcscId::Acsc1(Channel::ChannelOne),
                     cabin_zones,
-                    vec![
-                        ElectricalBusType::DirectCurrent(1),
-                        ElectricalBusType::AlternatingCurrent(1),
+                    [
+                        [
+                            ElectricalBusType::AlternatingCurrent(1), // 103XP
+                            ElectricalBusType::DirectCurrent(1),      // 101PP
+                        ],
+                        [
+                            ElectricalBusType::AlternatingCurrent(2),  // 202XP
+                            ElectricalBusType::DirectCurrentEssential, // 4PP
+                        ],
                     ],
                 ),
                 AirConditioningSystemController::new(
                     context,
                     AcscId::Acsc2(Channel::ChannelOne),
                     cabin_zones,
-                    vec![
-                        ElectricalBusType::DirectCurrent(2),
-                        ElectricalBusType::AlternatingCurrent(2),
+                    [
+                        [
+                            ElectricalBusType::AlternatingCurrent(2), // 101XP
+                            ElectricalBusType::DirectCurrent(2),      // 103PP
+                        ],
+                        [
+                            ElectricalBusType::AlternatingCurrent(2), // 204XP
+                            ElectricalBusType::DirectCurrent(2),      // 206PP
```

Comment:
I think there is a copy paste error here, right?

Reply:
Yeah this doesn't sound correct, 101XP and 103PP should be AC and DC bus 1, not 2

Reply:
Yes! I blame the docs 😄 . Fixed now so ACSC2 is powered by AC/DC 2 for lane 1 and AC/DC 2 for lane 2.